cup_of_join.py

```diff
@@ -0,0 +1,23 @@
+
+def join(*lists, sep="-"):
+    # This function receives lists and sep sign
```

Comment:
![Capture](https://user-images.githubusercontent.com/13467749/159524503-3b911815-bd44-4a9e-b2c7-3d52bf32bf91.JPG)

Reply:
this is how I want you to document functions in python